pkg/bwagreement/server_test.go

```diff
@@ -114,7 +118,7 @@ func NewTestServer(t *testing.T) *TestServer {
 }
 
 func newTestServerStruct(t *testing.T) *Server {
-	psqlInfo := getPSQLInfo()
+	psqlInfo := "postgres://postgres@localhost/pointerdb?sslmode=disable"
```

Comment:
Please see how it's done in https://github.com/storj/storj/blob/master/storage/postgreskv/client_test.go#L27

Reply:
I used visual code studio and you can run directly the test case with a mouse click and also implemented the support with a command line parameter as suggested.